src/js/popup/popup.js

```diff
@@ -161,6 +161,16 @@
       }
     },
     init: async () => {
+
+      const { isAndroid } = await browser.storage.local.get(
+        "isAndroid"
+      );
+
+      // Add CSS class for custom Firefox for Android styles to the panel
+      if (isAndroid) {
```

Comment:
Confirming the height fills the screen on android.

Reply:
It does not. I have #544 still out. We'll need to add extra CSS keyed off of the `is-android` classname. Width issue is fixed, though.

Reply:
Oops, I meant to say the width not the height 😵‍💫